product_docs/docs/pgd/5/manual/04-installing-software.mdx

```diff
@@ -0,0 +1,352 @@
+
+```
+
+The `max_worker_processes` value should be raised for larger clusters.
+To calculate the needed value see [Postgres configuration/settings](../postgres-configuration/#postgres-settings).
+
+#### Add a password to the Postgres enterprisedb user
+
+To allow connections between nodes, a password needs to be set on the Postgres enterprisedb user. 
+For this example, we are using the password `secret`.
+Select a different password for your deployments.
+You will need this password when we get to [Connecting the Cluster](05-connecting-cluster).
+
+```
+sudo -u enterprisedb psql edb -c "ALTER USER enterprisedb WITH PASSWORD 'secret'"
+
+```
+
+#### Enable inter-node authentication in pg_hba.conf
+
+Out of the box, Postgres allows local authentication and connections with the database but not external network connections. 
+To enable this, edit `pg_hba.conf` and add appropriate rules, including rules for the replication users.
+To simplify the process, use this command:
+
+```
+echo -e "host all all all md5\nhost replication all all md5" | sudo tee -a /var/lib/edb/as16/data/pg_hba.conf
+
+```
+
+It will append
+
+```
+host all all all md5
+host replication all all md5
+
+```
+
+to `pg_hba.conf` which will enable the nodes to replicate.
+
+#### Enable authentication between nodes
+
+As part of the process of connecting nodes for replication, PGD logs into other nodes. 
+It will perform that log in as the user that Postgres is running under.
+For epas, this is the `enterprisedb` user.
+That user will need credentials to log into the other nodes.
+We will supply these credentials using the `.pgpass` file which needs to reside in the user's home directory.
+The home directory for `enterprisedb` is `/var/lib/edb`.
+
+Run this command to create the file:
+
+```
+echo -e "*:*:*:enterprisedb:secret" | sudo -u enterprisedb tee /var/lib/edb/.pgpass; sudo chmod 0600 /var/lib/edb/.pgpass
+
+```
+
+### Restart the server
+
+After all these configuration changes, it is recommended that the server is restarted with:
+
+```
+sudo systemctl restart edb-as-16
+
+```
+
+#### Check the extension has been installed
+
+At this point, it is worth checking the extension is actually available and our configuration has been correctly loaded. You can query the pg_available_extensions table for the bdr extension like this:
+
+```
+sudo -u enterprisedb psql edb -c "select * from pg_available_extensions where name like 'bdr'"
+
+```
+
+Which should return an entry for the extension and its version.
+
+```
+ name | default_version | installed_version |                  comment
+------+-----------------+-------------------+-------------------------------------------
+ bdr  | 5.3.0           |                   | Bi-Directional Replication for PostgreSQL
+ ```
+
+You can also confirm the other server settings using this command:
+
+```
+sudo -u enterprisedb psql edb -c "show all" | grep -e wal_level -e track_commit_timestamp -e max_worker_processes
```

Comment:
Doing `SHOW ALL` and then grepping the result is really cumbersome. What's wrong with running
```
SHOW wal_level;
SHOW track_commit_timestamp;
SHOW max_worker_processes;
```